server/src/server.ts

```diff
@@ -179,11 +186,26 @@ class Server {
     this._modelValidationHandler = new ModelValidationHandler(this._modelValidationMiddleware);
     this._authHandler = new AuthHandler(this._authService);
     this._userHandler = new UserHandler(db);
-    this._recipeHandler = new RecipeHandler(db, this._recipeMiddleware);
+    this._recipeHandler = new RecipeHandler(db, this._recipeMiddleware, this._imgService);
     this._logHandler = new LogHandler();
   }
 
   /**
+   * UPLOADS
+   */
+  private _uploader (): void {
+    const storage = multer.diskStorage({
+      destination: (req, file, cb) => {
+        cb(null, Config.IMAGE_TMP_DIR);
```

Comment:
i would change this to:
```ts
`${process.cwd()}/${process.env.IMAGE_TMP_DIR || Config.IMAGE_TMP_DIR}`)
```
this creates a new folder into our server folder or looks into it. i think this is better!?